kernel-open/nvidia-uvm/uvm_kvmalloc.h

```diff
@@ -41,7 +41,9 @@
 //
 // This is in the header so callers can use it to inform their allocation sizes
 // if they wish.
-#define UVM_KMALLOC_THRESHOLD infinity
+//
+// Default value is 16. See: https://github.com/nanovms/nanos/blob/103518003874e7c8c3cb6e5a93d81fd0f197fde6/src/config.h#L84-L85
+#define UVM_KMALLOC_THRESHOLD 16
```

Comment:
The definition should use (1 << MAX_MCACHE_ORDER), and there is no need to reference the config.h lines, as they could change in the future, which would make this comment stale.